api/openapi-spec/swagger.json

```diff
@@ -5639,13 +5535,11 @@
         },
         "status": {
           "description": "Status of the condition, one of True, False, Unknown.",
-          "type": "string",
-          "default": ""
```

Comment:
What lead to the dropping of `"default"` here?

Reply:
whoops, new binary